src/DssSpellCollateralOnboarding.sol

```diff
@@ -31,60 +35,91 @@ contract DssSpellCollateralOnboardingAction {
+                kprFlatReward:         300,
+                kprPctReward:          10
+            })
+        );
+        DssExecLib.setStairstepExponentialDecrease(
+            MCD_CLIP_CALC_ETHSTETH_A,
+            90 seconds,
+            9900
+        );
+        DssExecLib.setIlkAutoLineParameters(
+            "CRVV1ETHSTETH-A",
+            3 * MILLION,
+            3 * MILLION,
+            8 hours
+        );
 
-        // DssExecLib.addNewCollateral(
-        //     CollateralOpts({
-        //         ilk:                   ,
-        //         gem:                   ,
-        //         join:                  ,
-        //         clip:                  ,
-        //         calc:                  ,
-        //         pip:                   ,
-        //         isLiquidatable:        ,
-        //         isOSM:                 ,
-        //         whitelistOSM:          ,
-        //         ilkDebtCeiling:        ,
-        //         minVaultAmount:        ,
-        //         maxLiquidationAmount:  ,
-        //         liquidationPenalty:    ,
-        //         ilkStabilityFee:       ,
-        //         startingPriceFactor:   ,
-        //         breakerTolerance:      ,
-        //         auctionDuration:       ,
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
+        // Extra for crop-join - need to authorize the join adapter with the cropper
+        DssExecLib.authorize(MCD_JOIN_ETHSTETH_A, CROPPER);
 
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
+        // Need to whitelist the stETH Medianizer with the wstETH Medianizer
+        DssExecLib.addReaderToWhitelistCall(WSTETH_MEDIAN, STETH_PRICE);
 
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
+        // Whitelist OSM - normally handled in addNewCollateral, but Curve LP Oracle format is not supported yet
+        DssExecLib.addReaderToWhitelistCall(CurveLPOracleLike(PIP_ETHSTETH).orbs(0), PIP_ETHSTETH);
+        DssExecLib.addReaderToWhitelistCall(CurveLPOracleLike(PIP_ETHSTETH).orbs(1), PIP_ETHSTETH);
 
         // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+        // Add the new clip and join to the Chainlog
+        DssExecLib.setChangelogAddress("CDP_REGISTRY", CDP_REGISTRY);
+        DssExecLib.setChangelogAddress("CROPPER", CROPPER);
```

Comment:
Not sure but maybe CROPPER should be `MCD_CROPPER`. We did that for `MCD_FLASH` and `MCD_IAM_AUTO_LINE`.

Also - consider adding `CROPPER_IMP`, as we probably want to bump a chainlog version if the implementation changes.

Reply:
done